### PR TITLE
docs: Update cmake documentation for experimental plugins.

### PR DESCRIPTION
**Pull request by brbzull0**

### CODE REVIEW

File: doc/developer-guide/plugins/example-plugins/index.en.rst [hunk 1]
```diff
@@ -48,9 +48,9 @@ understand the following topics:
 -  Working with HTTP header functions
 
 The two sample plugins discussed in this chapter are ``denylist_1.cc``
-and ``basic_auth.cc``. To build and install the example plugins use ::
-
-   ./configure --enable-example-plugins
+and ``basic_auth.cc``. To make this two plugins available, you must enable experimental plugins when
+building |TS| by passing the ``-DBUILD_EXPERIMENTAL_PLUGINS=ON`` to the ``cmake`` command
```
Comment (masaori335) [inline doc/developer-guide/plugins/example-plugins/index.en.rst:52]: This looks an "example" plugin. It should be `-DENABLE_EXAMPLE=ON`?

Comment (brbzull0) [reply doc/developer-guide/plugins/example-plugins/index.en.rst:52]: you are totally right! :cry:  Thanks!!

**Review by masaori335 — APPROVED** [review]
Looks good. Thanks!

**Comment by cmcfarlen** [comment]
Cherry-picked to v10.0.x

**Pull request merged**